labs/src/main/scala/org/scalalabs/basic/lab03/PatternMatchingExercise.scala

```diff
@@ -23,7 +24,14 @@ object PatternMatchingExercise {
    *************************************************************************/
 
   def describeLanguage(s: String): String = {
-    error("fix me")
+    s match {
+    	case "Java" | "Smalltalk" => "OOP"
```

Comment:
I’m pretty sure they aren’t boolean conditionals; I think they’re an aspect of pattern matching. Boolean conditionals only come in when you add `if` guards, I think.